rfcs/rfc00XX.md

```diff
@@ -0,0 +1,257 @@
+# Built-in lexical exporting
+
+## Preamble
+
+    Author:  Curtis Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      XX
+    Status:  Draft
```

Comment:
I suggest to follow [the proposed self-assigned ID scheme](https://github.com/Perl/RFCs/blob/main/docs/future.md#rfc-ids-and-how-to-self-assign) as long as this remains a draft: `OVID-XXXX`